docs/hub/datasets-daft.md

```diff
@@ -0,0 +1,79 @@
+# Daft
+
+[Daft](https://daft.ai/) is a high-performance data engine providing simple and reliable data processing for any modality and scale. Daft has native support for reading from and writing to Hugging Face datasets.
+
+<div class="flex justify-center">
+<img src="https://huggingface.co/datasets/huggingface/documentation-images/resolve/main/hub/daft_hf.png"/>
+</div>
+
+
+## Getting Started
+
+To get started, pip install `daft` with the `huggingface` feature:
+
+```bash
+pip install 'daft[hugggingface]'
```

Comment:
```suggestion
pip install 'daft[huggingface]'
```

Reply:
lol I didn't see it, thanks @davanstrien!